next.clayui.com/content/docs/css/components/navigation/base/index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I'm a bit scared about this here to many sub-sections until I get here, the menu gets pretty small... I think we can take the examples to stick together with `Application Bar`.

Reply:
Solved!